src/main/java/org/folio/converter/Field008RestoreFactory.java

```diff
@@ -0,0 +1,148 @@
+package org.folio.converter;
+
+import io.vertx.core.json.JsonObject;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Field008RestoreFactory {
+  private static final String ONE_SPACE =     " ";
+  private static final String TWO_SPACES =    "  ";
+  private static final String THREE_SPACES =  "   ";
+  private static final String FOUR_SPACES =   "    ";
+  private static final String FIVE_SPACES =   "     ";
+  private static final String SIX_SPACES =    "      ";
+  private static final String ELEVEN_SPACES = "           ";
+
+  private static EnumMap<ContentType, Field008RestoreStrategy> map = new EnumMap(ContentType.class);
+
+  private Field008RestoreFactory(){}
+
+  static {
+    map.put(ContentType.BOOKS, jsonObject ->
+      restoreHeader(jsonObject)
+        .concat(arrayToString("Ills", jsonObject))
+        .concat(jsonObject.getString("Audn"))
+        .concat(jsonObject.getString("Form"))
```

Comment:
I see these strings like "Form" or "Audn" are used several times, so they could be moved into constatnt variables